client/src/Movies/Movie.js

```diff
@@ -1,30 +1,30 @@
-import React, { useState, useEffect } from 'react';
-import axios from 'axios';
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import MovieCard from "./MovieCard";
+
+const Movie = props => {
+  const [movie, setMovie] = useState();
```

Comment:
what is the effect of removing the empty object from state?

Reply:
Im not 100% on this one, but removing the empty object allowed my map to work over the array of objects. So I believe since it had an empty object it was expecting an Object to be used for the states.